src/pages/steps/Step1.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement } from 'react'
+import { Title, InformationBox } from 'kyb-core'
+import DefaultTemplate from '../templates/DefaultTemplate'
+import DynamicFormattedMessage from '../../components/common/DynamicFormattedMessage'
+
+function Step1(): ReactElement {
+  return (
+    <DefaultTemplate>
+      <div>
+        <Title title="Société">
+          <DynamicFormattedMessage
```

Comment:
```suggestion
          <a href="https://www.google.com">{
            <DynamicFormattedMessage id="step1.tooltip" />
          </a>
```
This is way more understandable.
And change 
"step1.tooltip": "`<a>`Pourquoi est-ce important?`</a>`"
to 
"step1.tooltip": "Pourquoi est-ce important?"
in [src/lang/fr.json](src/lang/fr.json)

Avoid to use tags into translations

Reply:
Done